### DISCUSSION TITLE
I lost my enterprise username for 90 days

### DISCUSSION THREAD

**Discussion by wspada-godaddy** [community/community#140906] [Enterprise]
### Select Topic Area

Question

### Body

I was running in a problem when try to join into an organization where I joined into the organization with my personal account (wrong), got success, though, it was the incorrect way. After that, I left from the organization with my personal account and attempted the same process with my enterprise account (correct) but we got a problem with:

"There was an issue joining the enterprise: Your GitHub user account @wspada-godaddy is currently unlinked. However, you are attempting to authenticate with your Identity Provider using the '[wspada@godaddy.com](mailto:wspada@godaddy.com)' SAML identity which is already linked to a different GitHub user account in the organization. Please reach out to one of your GitHub organization owners for assistance.".

Then I reached out my organization members and we haven't gotten success on joining on the organization again. The last part was search something about that error and then I found this discussion "https://github.com/orgs/community/discussions/109307" where it says the simplest way is deleting the account and retry it again.... Though, I forgot the fact I have to wait 90 days to use the same username and I need to use this username now...

Please, could you assistance me? I need this username "wspada-godaddy" available as soon as we can.

**Comment by queenofcorgis** [comment]
Hey there @wspada2-godaddy 👋🏼 

I looked into this and spoke to som teammates and this issue would need to be solved through a support ticket. Would you mind opening a ticket on our [Support page](https://support.github.com/contact) please?
